infra/base-images/base-builder/indexer/clang_wrapper.py

```diff
@@ -245,10 +247,11 @@ def read_cdb_fragments(cdb_path: str) -> Any:
 
 
 def get_index_files(index_db_path) -> Iterator[str]:
+  """Get files referenced in the index."""
   conn = sqlite3.connect(index_db_path)
   cursor = conn.cursor()
 
-  query = f"""
+  query = """
         SELECT DISTINCT dirname, basename
         FROM location
     """
```

Comment:
Do we need a docstring for this short command? Docstring may add harmless but unnecessary spaces to the command, e.g.,
```python
'\n      SELECT DISTINCT dirname, basename\n      FROM location\n  '
```